### PR TITLE
(#215) Add note to online docs about using local admin for setup

### PR DESCRIPTION
**Pull request by steviecoaster**

### CODE REVIEW

**Comment by vexx32** [comment]
@steviecoaster we'll need to update the online version of the desktop readme as well.

File: QuickDeployment/V2/QuickDeploymentSetup.md [hunk 1]
```diff
@@ -303,6 +303,10 @@ A version of this readme file can be found in the [[Quick Deployment Desktop Rea
 > The online version is likely more up to date than the ReadMe you will find on the desktop (not including redacted items like credentials).
 > If there are conflicts between the desktop readme and what you see online, prefer the online version.
 
+:memo: **Note**
+
+When running the `Set-QDEnvironment` script ensure you are logged into the QDE VM with the _local admininistrator_ account. This is necessary for SQL to work properly initially.
+
```
Comment (vexx32) [inline QuickDeployment/V2/QuickDeploymentSetup.md:309]: Not really a fan of stacking successive note blocks one after another... maybe make this a part of the previous one?

```suggestion
> The online version is likely more up to date than the ReadMe you will find on the desktop (not including redacted items like credentials).
> If there are conflicts between the desktop readme and what you see online, prefer the online version.
>
> When running the `Set-QDEnvironment` script ensure you are logged into the QDE VM with the _local admininistrator_ account.
> This is necessary for SQL configuration to work properly initially.

```

**Review by vexx32 — CHANGES_REQUESTED** [review]

File: QuickDeployment/V2/QuickDeploymentSetup.md (hunk 1 above)
Comment (steviecoaster) [reply QuickDeployment/V2/QuickDeploymentSetup.md]: done!

**Comment by steviecoaster** [comment]
> @steviecoaster we'll need to update the online version of the desktop readme as well.

This is done as well in this commit.

File: QuickDeployment/V2/QuickDeploymentDesktopReadme.md [hunk 2]
```diff
@@ -90,6 +90,10 @@ If you would like to change the hostname of this VM, please do so **before** goi
 
 ### 3. Run the Set-QDEnvironment.ps1 Script
 
+:memo: **Note**
```
Comment (vexx32) [inline QuickDeployment/V2/QuickDeploymentDesktopReadme.md:93]:
```suggestion
> :memo: **Note**
>
```

Comment (steviecoaster) [reply QuickDeployment/V2/QuickDeploymentDesktopReadme.md]: Thank you. Messing with TOC, Ctrl+z'd too many times :joy:

File: QuickDeployment/V2/QuickDeploymentDesktopReadme.md [hunk 3]
```diff
@@ -90,6 +90,11 @@ If you would like to change the hostname of this VM, please do so **before** goi
 
 ### 3. Run the Set-QDEnvironment.ps1 Script
 
+>:memo: **Note**
```
Comment (vexx32) [inline QuickDeployment/V2/QuickDeploymentDesktopReadme.md:93]: Blockquote `>` needs a space after it or it'll break the Markdown formatting.

Comment (steviecoaster) [reply QuickDeployment/V2/QuickDeploymentDesktopReadme.md]: Done. This better be the last one :joy:

**Review by vexx32 — APPROVED** [review]

**Pull request merged**